app/editor/src/features/admin/reports/components/ReportSectionContent.tsx

```diff
@@ -97,6 +97,18 @@ export const ReportSectionContent = ({ index }: IReportSectionContentProps) => {
           label="Show Image"
           tooltip="Display the image for each content item in this section (if there is an image)"
         />
+        <Checkbox
```

Comment:
Using a Checkbox instead of a FormikCheckbox because undefined must remain true.